source/tyr/migrations/versions/465a7431358a_add_import_ntfs_to_mimir.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: 465a7431358a
+Revises: 3acdde6329db
+Create Date: 2018-02-23 14:50:23.086805
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '465a7431358a'
+down_revision = '3acdde6329db'
```

Comment:
I think you need to rebase, and check if your migration still work, new head is `242138b08c92`